src/datajudge/utils.py

```diff
@@ -0,0 +1,8 @@
+def check_module_installed(module_name: str) -> bool:
+    import importlib
+
+    try:
+        mod = importlib.import_module(module_name)
+        return mod is not None
```

Comment:
Why not just return `True`? Or can it happen that the module is not found and still not trigger the exception?

Reply:
Nope, just checked, it always throws an error :)